TheraLang.Web/Startup.cs

```diff
@@ -70,6 +74,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMainContext(Configuration.GetConnectionString("DefaultConnection"));
             services.AddUnitOfWork();
+            services.AddAuthentication(Configuration);
```

Comment:
Don't overload built-in functions. Change name so that its clear its our authentication